projects/plugins/jetpack/extensions/shared/test/block-fixtures.js

```diff
@@ -94,6 +95,9 @@ export default function runBlockFixtureTests( blockName, blocks, fixturesPath )
 					/* eslint-enable no-console */
 				}
 
+
+
+				const validationIssues = gatherValidationIssues( blocksActual );
```

Comment:
```suggestion
				const validationIssues = gatherValidationIssues( blocksActual );
```

Reply:
Thanks but I'm not gonna restart these tests right before merge just to remove two blank lines :) but I will apply this change in my next PR as a cleanup.

Reply:
I don't see "Ready to Merge" label, that's why I thought it was still under review.